StringOps.java

```diff
@@ -22,21 +22,112 @@ public class StringOps {
     //////                                               ///////
     ////////////////////////////////////////////////////////////
     public static void main(String[] args) {
-        
+        System.out.println(capVowelsLowRest("Hello World"));
+        camelCase("   Intro  to   coMPUter    sCIEncE ");
+        allIndexOf("Hello world",'l'); // output: {2, 3, 9}
     }
 
     public static String capVowelsLowRest (String string) {
-        // Write your code here:
-        return "";
+        String vowels = "aeiou";
+        String capVowels = "AEIOU";
+        String withBigVowels = "";
+        boolean isVowel = false;
+        for (int i = 0; i < string.length(); i++) {
+            for (int j = 0; j < vowels.length(); j++) {
+                if (string.charAt(i) == vowels.charAt(j)) {
+                    withBigVowels += (char) (string.charAt(i) - 32);
+                    isVowel = true;
+                } else if (string.charAt(i) == capVowels.charAt(j)) {
+                    withBigVowels += string.charAt(i);
+                    isVowel = true;
+                }
+            }
+            if ((string.charAt(i) >= 'A' && string.charAt(i) <= 'Z') && !isVowel) {
+                withBigVowels += (char) (string.charAt(i) + 32);
+            } else if (!isVowel) {
+                withBigVowels += string.charAt(i);
+            }
```

Comment:
Now, since we removed the for loop and we have `if` conditions to search vowels, there's no need to remember it already added, so we can use `else` before this `if`, than the last `else` will be fallback when all conditions are false.
```suggestion
            else if ((string.charAt(i) >= 'A' && string.charAt(i) <= 'Z') && !isVowel) {
                withBigVowels += (char) (string.charAt(i) + 32);
            } else  {
                withBigVowels += string.charAt(i);
            }
```